### ISSUE TITLE
How to use Prisma indexes as mixin?

### ISSUE THREAD

**Issue by vineryap** [cwqt/refract#10]
Hi, I see that the library supports mixins but I can't find an example of using it with Prisma indexes `@@index` or other keywords that start with `@@`.
Is it possible to put Prisma indexes in a separate file and merge it later?

Cheers!

**Comment by vineryap** [comment]
Found it!
Sorry, I didn't pay enough attention and test more.
It can be done using `.Raw()`

**Comment by cwqt** [comment]
my bad for missing `.Block()` on Mixins - this has been added in https://github.com/cwqt/refract/commit/b2c56aa1b560e1a862069f971074928e10324b69 & released with [1.3.3](https://www.npmjs.com/package/@cwqt/refract/v/1.3.3)

```ts
import { Compound } from "@cwqt/refract"

const Test = Mixin()
  .Block(Compound.Index(["mixin", "index"]))
```

Which'll give you `@@index([mixin, index])`
